ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -101,7 +101,7 @@ var comparisonClass = 'Comparison',
       'RPAREN',
     ],
     BETWEEN: ['VALUE'],
-    IS_NULL: ['END'],
+    IS_NULL: ['RPAREN', 'LOGICAL', '[', ']'],
```

Comment:
This is used to parse the query and place each part of the query on a stack. 
For example the query `("location" IS NULL)` has several parts (left and right parens, the operator, attribute), so I'm just just tying the IS_NULL object to contain such parts since the regex handles the splitting of the query.